### PR TITLE
Version 0.2.3

### PR DESCRIPTION
**Pull request by BSchilperoort**
I decided to bump the version so people can use the latest fixes.

@kjdoore I added you to the CITATION.cff file, can you check if I found the correct ORCiD?

### CODE REVIEW

File: CITATION.cff [hunk 1]
```diff
@@ -14,14 +14,19 @@ authors:
     family-names: Liu
     orcid: 'https://orcid.org/0000-0002-1966-8460'
     affiliation: Netherlands eScience Center
+- given-names: Keith
```
Comment (kjdoore) [inline CITATION.cff:17]: Yeah, this is correct! Thanks for adding me!

(Looks like this line lost its indent.)

```suggestion
   - given-names: Keith
```

File: CITATION.cff [hunk 2]
```diff
@@ -14,14 +14,19 @@ authors:
     family-names: Liu
     orcid: 'https://orcid.org/0000-0002-1966-8460'
     affiliation: Netherlands eScience Center
+   - given-names: Keith
+    family-names: Doore
+    orcid: 'https://orcid.org/0000-0001-5035-4016'
+    affiliation: United States Geological Survey
```
Comment (kjdoore) [inline CITATION.cff:20]:
```suggestion
  - given-names: Keith
    family-names: Doore
    orcid: 'https://orcid.org/0000-0001-5035-4016'
    affiliation: United States Geological Survey
```

Comment (kjdoore) [inline CITATION.cff:20]:
```suggestion
  - given-names: Keith
    family-names: Doore
    orcid: 'https://orcid.org/0000-0001-5035-4016'
    affiliation: United States Geological Survey
```

File: CITATION.cff (hunk 1 above)
Comment (BSchilperoort) [reply CITATION.cff]: well spotted :)

**Review by kjdoore — APPROVED** [review]
After my several attempts to fix a single indent, this looks good to go!

**Comment by BSchilperoort** [comment]
> After my several attempts to fix a single indent, this looks good to go!

Thanks for fixing it @kjdoore 

The new version is now on pypi https://pypi.org/project/xarray-regrid/

**Pull request merged**